x-pack/legacy/plugins/uptime/public/components/functional/charts/donut_chart.tsx

```diff
@@ -82,6 +92,7 @@ export const DonutChart = ({ height, down, up, width }: DonutChartProps) => {
           width={width}
           height={height}
         />
+        {down === 0 && <GreenCheckIcon className="greenCheckIcon" type="checkInCircleFilled" />}
```

Comment:
had to use className because of some mysterious reason styled component not being able to found in Enzye test so using class selector.

Reply:
We may want to include an actual comment in the code explaining this.

Reply:
done